src/components/shared/transactionSignature/index.js

```diff
@@ -0,0 +1,24 @@
+// istanbul ignore file
+import { withRouter } from 'react-router';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { multisigTransactionSigned, transactionDoubleSigned } from '@actions';
+import { withTranslation } from 'react-i18next';
+import TransactionSignature from './transactionSignature';
+
+const mapStateToProps = state => ({
+  activeToken: state.settings.token.active,
```

Comment:
Active token doesn't seem to be used in wrapped component and if so, should be removed